packages/mobile/src/i18n/i18nSlice.ts

```diff
@@ -0,0 +1,60 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { Actions as AppActions, UpdateConfigValuesAction } from 'src/app/actions'
+import { REMOTE_CONFIG_VALUES_DEFAULTS } from 'src/firebase/remoteConfigValuesDefaults'
+import { getRehydratePayload, REHYDRATE, RehydrateAction } from 'src/redux/persist-helper'
+
+export interface State {
+  language: string | null
+  allowOtaTranslations: boolean
+  otaTranslationsLastUpdate: number
+  otaTranslationsAppVersion: string
+  otaTranslationsLanguage: string
+}
+
+const initialState: State = {
+  language: null,
+  allowOtaTranslations: REMOTE_CONFIG_VALUES_DEFAULTS.allowOtaTranslations,
+  otaTranslationsLastUpdate: 0,
+  otaTranslationsAppVersion: '0',
+  otaTranslationsLanguage: '',
+}
+
+export interface OtaTranslationsUpdatedAction {
+  otaTranslationsLastUpdate: number
+  otaTranslationsAppVersion: string
+  otaTranslationsLanguage: string
+}
+
+export const i18nSlice = createSlice({
```

Comment:
Should we just name is `slice`? and same for the file?

Reply:
nice, i updated this.

i find generic names a bit harder to work with when there are many files open (e.g. when they all say reducer.js in the tab title) so was a natural reflex that i did the specific naming like this :/ but indeed consistency is key and we already have the generic naming pattern in the project so we should keep this :)